Model/Indexer/CategoryObserver.php

```diff
@@ -40,12 +54,25 @@ public function __construct(IndexerRegistry $indexerRegistry, ConfigHelper $conf
     public function beforeSave(CategoryResourceModel $categoryResource, CategoryModel $category)
     {
         $categoryResource->addCommitCallback(function() use ($category) {
-            if (!$this->indexer->isScheduled() || $this->configHelper->isQueueActive()) {
+            $collectionIds = [];
+            // To reduce the indexing operation for products, only update if these values have changed
+            if ($category->getOrigData('name') !== $category->getData('name')
```

Comment:
Hey @bsuravech 

I think, it would make sense to add an extra condition to take into account category path to support moving properly moving category to another category.

Regards,
Maxim

Reply:
Hey @maximbaibakov! Good catch! Updating now!